servers/visual/visual_server_scene.cpp

```diff
@@ -2430,7 +2430,11 @@ bool VisualServerScene::_render_reflection_probe_step(Instance *p_instance, int
 		}
 
 		_prepare_scene(xform, cm, false, RID(), VSG::storage->reflection_probe_get_cull_mask(p_instance->base), p_instance->scenario->self, shadow_atlas, reflection_probe->instance);
+
+		bool forced_sync_backup = VSG::storage->is_forced_sync_shader_compile_enabled();
+		VSG::storage->set_forced_sync_shader_compile_enabled(true);
```

Comment:
Is this left in from debugging? Or is there a reason that reflection probes always need force sync enabled?

Reply:
It _was_ intended because in the project this was initially written for reflection probes were `UPDATE_ONCE` and so they only have one chance to capture the look of the real shaders.

However, now I realize that for general use this is not enough. Maybe it's just a matter of doing that unless it's `UPDATE_ALWAYS`. Probably it will be more involved than that, but may be a good start.

Reply:
I'm wondering if it would be enough to change ``true`` to update_mode == UPDATE_MODE_ONCE.

A perfect solution would be to delay the capture until all shaders are compiled. But that is probably out of scope for this PR